src/lib/Util/adapters/ibm/index.js

```diff
@@ -0,0 +1,232 @@
+/**
```

Comment:
Let's remove these comments - we have some sample json files to refer to if needed.

Reply:
ok, removed across all the other adapters as well